src/components/index.js

```diff
@@ -12,6 +12,7 @@ import Chips from "./Chips/Chips";
 import Input from "./Input/Input";
 import Filter from "./Filter/Filter";
 import RadioInput from "./RadioInput/RadioInput";
+import RequiresAuth from "./RequiresAuth/RequiresAuth";
```

Comment:
Instead of importing and then exporting again, you can export it without importing 
`export default as ComponentName from "path/to/component"`

Reply:
Ok, thanks. didn't know that.